compose.yml

```diff
@@ -5,11 +5,13 @@ services:
     environment:
       - POSTGRES_USER=${POSTGRES_USER}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
-      - POSTGRES_DB=uinnovate_test_db # Default database's name
+      - POSTGRES_DB=${POSTGRES_DB_NAME} # Default database's name
```

Comment:
Would you be able to add the POSTGRES_DB_NAME to the .env.template please? :)

Reply:
Completely forgot. Done!